custom_components/daikin_onecta/daikin_api.py

```
@@ -69,93 +70,77 @@ async def async_get_access_token(self) -> str:
+                    if self.rate_limits["remaining_minutes"] > 0:
+                        ir.async_delete_issue(self.hass, DOMAIN, "minute_rate_limit")
+
+                    if self.rate_limits["remaining_day"] > 0:
+                        ir.async_delete_issue(self.hass, DOMAIN, "day_rate_limit")
+
+                    _LOGGER.debug("BEARER RESPONSE STATUS: %s", resp.status)
+
+                    if method == "GET" and resp.status == 200:
+                        try:
+                            return json.loads(data)
+                        except Exception:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)